nodes/src/nodes/tool_firecrawl/IInstance.py

```
@@ -26,23 +26,125 @@
+
+        result = firecrawl_wrapper(lambda: self.IGlobal.app.scrape(url))
+
+        content = getattr(result, 'markdown', None) or getattr(result, 'html', None) or ''
+        if not isinstance(content, str):
+            content = json.dumps(content)
+
+        metadata = getattr(result, 'metadata', None)
+        if metadata is not None and not isinstance(metadata, dict):
+            try:
+                metadata = metadata.model_dump(exclude_none=True) if hasattr(metadata, 'model_dump') else {}
+            except Exception:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)